package.json

```diff
@@ -135,6 +146,50 @@
     "nodemon-notifier-cli": "https://github.com/Slessi/nodemon-notifier-cli.git",
     "prettier": "^1.19.1",
     "ts-jest": "24.2.0",
+    "ts-node": "^8.0.0",
     "typescript": "^3.8.2"
+  },
+  "oclif": {
+    "commands": "./build/commands",
+    "bin": "vtex",
+    "plugins": [
+      "@oclif/plugin-help",
+      "@tiagonapoli/oclif-plugin-spaced-commands"
+    ],
+    "hooks": {
+      "init": "./build/oclif/hooks/init"
+    },
+    "topics": {
+      "config": {
```

Comment:
Should we add `Run 'vtex something --help' to see all subcommands`? We know though that in these cases `--help` can be suppressed, but then maybe we start encouraging the users to use this flag

Reply:
I don't have a strong opinion about this. I removed the `--help` flag after a feedback from @diegoximenes that commented about the flag being unnecessary.